index.html

```diff
@@ -76,5 +76,9 @@ <h4 class="logo">Relaxr</h4>
 			<p class="copyright">Copyright 2015. Relaxr.</p>
 			<script src="http://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
 		</footer>
+
+		<script src="http://code.jquery.com/jquery-latest.min.js"></script>
+
+		<script src="/Users/bnicolemcgowan/Desktop/General Assembly/FEWD31/Week 06/B. Nicole McGowan Assignment 4 Homework/FEWD31-Assignment4/jquery_dom_selector_practice/js/index.js"></script>
```

Comment:
This needs to be a relative path. Currently you have it as an "absolute" path. It will only work on your machine. Instead it should simply be

```html
<script src="jquery_dom_selector_practice/js/index.js"></script>
```